### PR TITLE
Import CSV Usage in PyAdditive/Examples

### PR DESCRIPTION
**Pull request by safeerehman**
- Update unit tests under `tests_*.py` to check for nan values when creating a simulation input. 
- Update pyadditive/examples to show the usage of import csv file into a study. 
- Update pyproject.toml/doc section

### CODE REVIEW

File: tests/test_microstructure.py [hunk 1]
```diff
@@ -331,6 +331,37 @@ def test_MicrostructureInput_setters_raise_ValueError_for_values_out_of_range():
...
+        input.cooling_rate = float("nan")
+    with pytest.raises(ValueError, match="thermal_gradient must be a number"):
+        input.thermal_gradient = float("nan")
+    with pytest.raises(ValueError, match="melt_pool_width must be a number"):
+        input.melt_pool_width = float("nan")
+    with pytest.raises(ValueError, match="melt_pool_depth must be a number"):
+        input.melt_pool_depth = float("nan")
+    with pytest.raises(ValueError, match="random_seed must be a number"):
+        input.random_seed = float("nan")
+
```
Comment (pkrull-ansys) [inline tests/test_microstructure.py:363]: This is ok but there's a common idiom in python that uses the `setattr()` function to set an object's attribute by name. For example, the body of the test could be reduced to a couple lines.
```
with pytest.raises(ValueError, match=field + " must be a number"):
    setattr(input, field, float("nan"))
```
where `field` is a string containing the name of the attribute you're setting. You'd need to use `@pytest.mark.parametrize` and put the attribute names in a list. `field` would be the parameter passed into the test function.

**Review by pkrull-ansys — CHANGES_REQUESTED** [review]
Bump the version in `pyproject.toml` and add a release note saying you updated the example.

File: examples/11_advanced_parametric_study.py [hunk 2]
```diff
@@ -146,14 +146,26 @@ class provides data management and visualization features that ease the task.
...
 study.data_frame().to_csv("demo-study.csv")
 
+###############################################################################
+# Import a study from a CSV file
+# ------------------------------
+# Import a study from a CSV file using the :meth:`ParametricStudy.import_csv_study` method.
+# The CSV file must contain the same columns as the parametric study data frame.
+# The :meth:`ParametricStudy.import_csv_study` method will return a list of errors for each
+# simulation that failed to import. All other valid simulations will be added to the study.
+
```
[the remark this answers is not in the document]
Comment (safeerehman) [reply examples/11_advanced_parametric_study.py:156]: Yes, good point, I will update the append the error list to also include the number of duplicates removed.

File: src/ansys/additive/core/parametric_study/parametric_study.py [hunk 3]
```diff
@@ -1615,7 +1615,7 @@ def add_simulations_from_data_frame(self, df: pd.DataFrame) -> list[str]:
         """
 
         # check valid inputs
-        drop_indices, error_list = [], []
+        drop_indices, error_list, duplicates = list(), list(), int()
```
Comment (pkrull-ansys) [inline src/ansys/additive/core/parametric_study/parametric_study.py:1618]:
```suggestion
        drop_indices, error_list = list(), list()
        duplicates = 0
```

**Review by pkrull-ansys — APPROVED** [review]
Approved with a suggestion.

**Comment by safeerehman** [comment]
Closing due to unsigned commit
Duplicate PR at #259

**Pull request closed without merging**